build-scripts/generateRunners.ts

```diff
@@ -0,0 +1,66 @@
+/*!
+ * Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: MIT
+ */
+
+import * as fs from 'fs'
+import * as path from 'path'
+
+interface RunnerGenerator {
+    TaskName: string,
+    TaskClient: string,
+    SetResult?: string
```

Comment:
`SetResult` lacks meaning in a schema. It is ambiguous too, since there is a success and failure result. 
Consider something like `successResult`

Reply:
successResult is fine